spec/support/feature_spec_helper.rb

```diff
@@ -10,4 +10,9 @@ def login_as(user)
     setup_mock_auth(:myusa, user)
     visit '/auth/myusa'
   end
+
+  def focus_field(field_id)
```

Comment:
since we'er only using this method once right now, I'd suggest putting it in the spec file where it is being used. Easier to check it out that way!